src/angularjs/src/app/analysis-jobs/import/analysis-jobs-import.controller.js

```diff
@@ -14,7 +14,7 @@
         var ctl = this;
 
         function initialize() {
-            ctl.neighborhoods = Neighborhood.all().$promise.then(function(data) {
+            ctl.neighborhoods = Neighborhood.all({ordering: 'label'}).$promise.then(function(data) {
```

Comment:
That would affect the "Neighborhood" admin list, too.  Actually, looking at that list, it seems to be expecting the neighborhoods to be sorted by descending creation date (the page buttons are "Older" and "Newer") but there's no `ordering` property, so they're not sorted by creation date or by label.  So I added an `ordering` property but set it to `-created_at` for the benefit of the neighborhood list.  It would be nice to enable other sorting and filtering options on the Neighborhoods page, but for now I think this is slightly better.